packages/snaps-controllers/src/services/iframe/IframeExecutionService.test.browser.ts

```diff
@@ -50,9 +50,22 @@ describe('IframeExecutionService', () => {
     const iframe = document.querySelector('iframe');
     assert(iframe);
 
-    expect(iframe.contentDocument).toBeNull();
-    expect(() => iframe.contentWindow?.document).toThrow(
-      /Blocked a frame with origin ".+" from accessing a cross-origin frame./u,
+    await new Promise((resolve) => setTimeout(resolve, 1000));
+
+    const message = new Promise((resolve) => {
+      window.addEventListener('message', (event) => {
+        resolve(event.data);
+      });
+    });
+
+    // Creates an iframe attempts to access the iframe created by the execution
+    // service. This should fail due to the sandboxing.
+    const testFrame = document.createElement('iframe');
+    testFrame.src = 'http://localhost:4567/test/sandbox';
```

Comment:
is there a way to get the `localhost:4567` part from some shared variable somewhere? it's just very specific and i want it to adjust automatically in case the port changes or something